CalorieTracker/CalorieTracker/AppDelegate.swift

```diff
@@ -0,0 +1,93 @@
+//
+//  AppDelegate.swift
+//  CalorieTracker
+//
+//  Created by Kat Milton on 8/23/19.
+//  Copyright © 2019 Kat Milton. All rights reserved.
+//
+
+import UIKit
+import CoreData
+
+@UIApplicationMain
+class AppDelegate: UIResponder, UIApplicationDelegate {
+
+    var window: UIWindow?
+
+
+    func application(_ application: UIApplication, didFinishLaunchingWithOptions launchOptions: [UIApplication.LaunchOptionsKey: Any]?) -> Bool {
+        // Override point for customization after application launch.
+        return true
+    }
+
+    func applicationWillResignActive(_ application: UIApplication) {
```

Comment:
For future apps, remember to try to clean up extra unused function calls when you can.